src/overlap_check/ProgressBar.cpp

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
Good point. I looked at that a bit, yeah. I saw that it adds 10-15% to the runtime. I think the additional time is worth the indication of progress to our users - especially given how long it could take for the check to complete.